.github/pull_request_template.md

```diff
@@ -1,12 +1,16 @@
-# Description
+## Description
 
 <!-- Please write a summary of your changes and why you made them.-->
```

Comment:
Will these comments be visible in the commit message body?

Reply:
Nope. See https://docs.mergify.com/configuration/#template:

> By default, the HTML comments are stripped from body. To get the full body, you can use the body_raw attribute.